docs/docs/team.md

```diff
@@ -1,4 +1,7 @@
-# :material-account-supervisor-circle-outline: Team 
+import {interpolate} from '@docusaurus/Interpolate';
+const feedback = interpolate('https://github.com/h2oai/docs-issues-requests/issues/new?assignees=5675sp&labels=area%2Fdocumentation%2C+h2o-health%2C+haic-app&body=%23%23%23%20Documentation%20issue%2Frequest%0A%0A%3C!--%20Please%20provide%20a%20clear%20and%20concise%20description%20of%20the%20documentation%20issue%2Frequest%20--%3E%0A%0A%23%23%23%20Additional%20context%0A%0A%3C!--%20Please%20add%20any%20other%20context%20about%20the%20issue%2Frequest%20here%20(e.g.%2C%20images)%20--%3E%0A%0A%23%23%23%20Page%20details%20%0A%0A-%20Application%20name%3A%20H2O%20Health%0A-%20Application%20version%3A%20{version}%0A-%20Page%20title%3A%20{title}%20&title=%5BHAIC-APP%5D', {title: metadata.permalink, version: metadata.version});
+
+# Team 
```

Comment:
@5675sp @oshi98 are we adding this "Team" page in all docs or is it something specific to H2O Health?

Reply:
Not in all docs but in H2O Health, there were [Team pages](https://docs.h2o.ai/h2o-health/team/) with developers' names. Maybe we can check with the dev team if it's necessary. If not, we can remove it from the new docs. WDYT?

Reply:
its specific to H2O Health @sherenem